nibabel/nifti1.py

```diff
@@ -1933,6 +1934,40 @@ def set_sform(self, affine, code=None, **kwargs):
             else:
                 self._affine[:] = self._header.get_best_affine()
 
+    def transpose(self, ornt):
+        """Apply an orientation change and return a new image
+
+        If image already has orientation, return the original image, unchanged
```

Comment:
You're right. I wrote that bit while I thought `ornt` was the target orientation, not the transformation.